week-2/Homework/F-logical-operators/exercise2.js

```diff
@@ -5,8 +5,33 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  if (number < 0) {
+    return true;
+  }
+  return false;
+}
 
+function isBetween5and10(number) {
+  if (number <= 5 && number >= 10) {
+    return true;
+  }
+  return false;
+}
+
+function isShortName(name) {
+  if (name === "Daniel") {
+    return true;
+  }
+  return false;
+}
+
+function startsWithD(name) {
+  if (name[0] === "D") {
+    return true;
+  }
+  return false;
+}
```

Comment:
Correct. However, you could write the each of the bodies for the above functions in one line, without needing to use an if block. Think about what the expression you are using for the if conditional evaluates to, and what you need the function to return.

Reply:
Thank a lot. I understand what you tell about. I will change the code in order to do it better. Thank you.